.github/workflows/_e2e.yaml

```diff
@@ -104,8 +104,13 @@ jobs:
           EOF
           sudo service docker restart
       - name: "GitHub Actions: run E2E tests"
+        id: e2e
         if: steps.eval-params.outputs.run-type == 'github'
         run: |
+          FILENAME="stats$RANDOM.txt"
```

Comment:
weirdly enough looks like only 1 artifact is created so I guess RANDOM is not that random...